package j$.util.stream;

/* loaded from: classes2.dex */
final class t extends z {
    public final /* synthetic */ int m;
    final /* synthetic */ Object n;

    /* JADX WARN: 'super' call moved to the top of the method (can break code semantics) */
    public /* synthetic */ t(b bVar, int i, Object obj, int i2) {
        super(bVar, i, 1);
        this.m = i2;
        this.n = obj;
    }

    @Override // j$.util.stream.b
    final d2 w0(int i, d2 d2Var) {
        switch (this.m) {
            case 0:
                return new s(this, d2Var, 0);
            case 1:
                return new s(this, d2Var, 4);
            case 2:
                return new s(this, d2Var, 5);
            case 3:
                return new s(this, d2Var, 6);
            case 4:
                return new V(this, d2Var, 4);
            case 5:
                return new e0(this, d2Var, 3);
            case 6:
                return new n(this, d2Var, 6);
            default:
                return new Q1(this, d2Var);
        }
    }

    /* JADX WARN: 'super' call moved to the top of the method (can break code semantics) */
    public t(A a, j$.util.function.l lVar) {
        super(a, 0, 1);
        this.m = 3;
        this.n = lVar;
    }
}
